<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupWindow</class>
 <widget class="QMainWindow" name="SignupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="SignupLabel">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>20</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>1010</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="UsernameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>123</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>1010</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ConfirmPasswordLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfirmPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>1010</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FirstNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstNameLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>1010</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MiddleNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MiddleNameLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>1010</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LastNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastNameLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>1010</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="EmailLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>1010</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SignupButton">
    <property name="geometry">
     <rect>
      <x>839</x>
      <y>791</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>790</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
